docker/.env.example.full

```
@@ -343,6 +343,10 @@ ILIVEDATA_PID=""
 # Secret value
 ILIVEDATA_SECRET_KEY=""
 
+# Timeout in seconds for ILIVEDATA text moderation requests
+# (Optional - default: 5)
+ILIVEDATA_TIMEOUT_SECONDS="5"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)